README.adoc

```diff
@@ -157,18 +157,18 @@ have IPv4 addresses, the address on eth2 is used.
 === HTTP Proxy Settings
 
 In an environment where the HTTP traffic needs to pass through an
-HTTP proxy server, set the `http_proxy`, `http_proxy_user` and
-`http_proxy_password` proxy configurations in the Vagrantfiles to enable
+HTTP proxy server, set the `proxy`, `proxy_user` and
```

Comment:
@budhrg @hferentschik after thinking little bit more, I think we should mention deprecation of `http_` prefix in the documentation [1] in this PR and also update [2]

[1] https://github.com/projectatomic/vagrant-service-manager#http-proxy-settings
[2] https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/installing.adoc#step-6-optional-set-up-adb-behind-an-http-proxy